docs/api/apiv3/tags/file_links.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: |-
+  TBD
```

Comment:
We should add something here at least when the feature has been developed.

Reply:
Acknowledged. But in this very early state I have the feeling, that every word I'd write in here, I'd need to replace in 2 weeks from here.